### ISSUE TITLE
Cannot remap Recipe Back button

### ISSUE THREAD

**Issue by billvardou** [GTNewHorizons/NotEnoughItems#143]
I have tried remapping the default Back Recipe button (which is bound to keyboard "BACK") to my mouse's physical back button. The button is not recognised when i press it at all and it defaults to "NONE". I have tried binding the same button to other default minecraft options, and as I observed at the config files, my mouse's button value is -97. I tried hard coding the value to the client.cfg config file of NEI, but it reverts to the default value of keyboard "BACK" button, which is 14. Any thoughts? Has anyone else encountered something similar?
![Screenshot_1](https://user-images.githubusercontent.com/63598727/126907024-779a33ac-7279-4610-9a06-bc4c3b3a510c.png)

**Comment by mitchej123** [comment]
I believe this to be a limitation of LWJGL.  You could look into an external program (auto hotkeys, or something of the like) that would map the mouse back button to a keyboard code that could be mapped.

**Comment by ryankhart** [comment]
Sorry to resurrect this long closed discussion, but could it not be possible to backport the way that JEI handles its keybinds by letting the user configure it inside the Minecraft "Options... -> Controls..." UI instead?

I just tested and confirmed that I was able to map the jump function to "Button 4" on my mouse and it lets me jump with that mouse button.

I understand that this greatly increases the scope of the project, but I've seen several other JEI features get backported to NEI in the past. Perhaps this idea could be put in the backlog, though.
